tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -689,10 +691,14 @@ public function theUserListsAllHisAvailableSpacesWithHeadersUsingTheGraphApi(
 	 * @throws Exception
 	 */
 	public function theUserListsAllAvailableSpacesUsingTheGraphApi(string $user, string $query = ''): void {
+		$password = $this->featureContext->getPasswordForUser($user);
+		if ($this->authAppContext->isUsingAuthAppToken()) {
+			$password = $this->authAppContext->getLastCreatedToken();
```

Comment:
can we implement this inside $this->featureContext->getPasswordForUser($user), so that automatically every api request gets benefit from it

Reply:
logic should be in `$this->featureContext->getPasswordForUser($user)` :+1: